waku/common/utils/nat.nim

```diff
@@ -54,6 +54,9 @@ proc setupNat*(natConf, clientId: string,
         endpoint.udpPort = some(extUdpPort)
 
   else: # NatNone
+    if natConf == "none":
```

Comment:
isnt this redundant? its the else case of `if strategy != NatNone:` so it will always be none?

Reply:
Yes, but before it executed the code below:

```nim
    if not natConf.startsWith("extip:"):
      return err("not a valid NAT mechanism: " & $natConf)
```

Reply:
Shall we just return an error in this case (` else: # NatNone `)?

I have the impression that the next is always run:
```nim
return err("not a valid NAT mechanism: " & $natConf)
```